logger.py

```diff
@@ -0,0 +1,143 @@
+"""Logging configs."""
+
+import logging
+import os
+from logging.handlers import RotatingFileHandler
+from typing import Tuple
+
+import bpy
+
+from . import colors, ui
+from .config import HANA3D_LOG_LEVEL, HANA3D_NAME
+from .report_tools import execute_wrapper
+
+
+def setup_logger(): # noqa WPS210,WPS213
+    """Logger setup."""
+    log_level = HANA3D_LOG_LEVEL
+    logger = logging.getLogger('')
+    logger.setLevel(logging.DEBUG)
+
+    dir_path = os.path.join(os.getcwd(), 'hana3d_logs')
```

Comment:
Se o blend não estiver salvo `os.getcwd()` vai retornar a pasta de instalação do Blender, que provavelmente no Windows vai esta no Program Files, então não vai ter permissão de escrita se não for executado como administrador.

Reply:
Acho que poderia salvar na pasta `hana3d_data` mesmo.